dashboard/src/components/entitypage/EntityPageView.js

```diff
@@ -306,8 +325,11 @@ const EntityPageView = ({ entity, setVersion, activeVersions }) => {
               aria-label="simple tabs example"
             >
               {showMetrics && <Tab label={"metrics"} {...a11yProps(0)} />}
+              {showStats && (
+                <Tab label={"stats"} {...a11yProps(showMetrics ? 1 : 0)} />
```

Comment:
You have this showMetrics ? 1 : 0 in a lot of places, make it a variable and make the 1 and 0 not be magic numbers